packit/api.py

```diff
@@ -430,8 +433,7 @@ def status(self):
         if builds:
             logger.info("\nLatest builds:")
             for build in builds:
-                koji_builds_str = "\n".join(f" - {b}" for b in builds[build])
-                logger.info(f"{build}:\n{koji_builds_str}")
+                logger.info(f"{build}:\n- {builds[build]}")
```

Comment:
`builds` is a dictionary so you should use `builds.items()` to walk through its keys:values

```python
  for branch, branch_builds in builds.items(): 
      print(f"{branch}: {branch_builds}")
```